### PR TITLE
Fix residual block

### PR DESCRIPTION
**Pull request by Tsarpf**
Hi,

Great repo, it's a good starting point for SR in pytorch.

I think every "Residual Block" is missing an activation function at the end, at least according to the original ResNet paper: 
image
(from https://arxiv.org/abs/1512.03385)

it probably makes sense to add it since every other convolution currently basically does two consecutive convolutions in a row which isn't useful(?)

### CODE REVIEW

**Comment by Tsarpf** [comment]
Okay apparently it's on purpose in the paper you're implementing here, and they cite this website http://torch.ch/blog/2016/02/04/resnets.html that has this explanation for skipping the last activation function:

We can take this philosophy one step further: should we remove the ReLU layers at the end of each residual block? ReLU layers also perturb data that flows through identity connections, but unlike batch normalization, ReLU’s idempotence means that it doesn’t matter if data passes through one ReLU or thirty ReLUs. When we remove ReLU layers at the end of each building block, we observe a small improvement in test performance compared to the paper’s suggested ReLU placement after the addition. However, the effect is fairly minor. More exploration is needed.

I guess never mind then.. :) It's as it should be now.
